audit/entry_formatter.go

```diff
@@ -623,3 +638,71 @@ func newTemporaryEntryFormatter(n *EntryFormatter) *EntryFormatter {
 func (s *nonPersistentSalt) Salt(_ context.Context) (*salt.Salt, error) {
 	return salt.NewNonpersistentSalt(), nil
 }
+
+// excludeFields takes an entry (RequestEntry or ResponseEntry) and attempts to
+// exclude fields that have been configured on the formatter node.
+func (f *EntryFormatter) excludeFields(entry any) (map[string]any, error) {
+	m := make(map[string]any)
+	err := mapstructure.Decode(entry, &m)
```

Comment:
By converting the input structure into a map first, are we at risk of breaking any existing formatting that relies on the `json` struct tags?  Maybe we should use DecoderConfig.TagName = "json" here too?